### ISSUE TITLE
arduino

### ISSUE THREAD

**Issue by hedva92** [manio/skymax-demo#7]
Hi, I'm trying to connect via arduino to the ivertor. Unfortunately, so far without success. Could I ask you for one raw command to read the firmware version, including the crc code? It should be something like this.. QVFW<CRC><CR> I tried to compute crc, send different variants and still nothing: / Port setup is classic, just 2400baud. Thanks for the help.

**Comment by manio** [comment]
Yes, QVFW should be for retrieving firmware. The response should be eg:
(VERFW:00072.70S§\r

But how is this related to skymax-demo?

**Comment by manio** [comment]
btw: you have a crc code generator in my sources...
